jupysql_plugin/_widgets.py

```diff
@@ -0,0 +1,67 @@
+from jupysql_plugin import __version__, _module_name
+
+from ipywidgets import DOMWidget
+from traitlets import Unicode
+
+
+class FormWidget(DOMWidget):
+    """
+    A sample widget that displays a form and process it. This widget is an example,
+    not intended for end-users
+    """
+
+    _model_name = Unicode("FormModel").tag(sync=True)
+    _model_module = Unicode(_module_name).tag(sync=True)
+    _model_module_version = Unicode(__version__).tag(sync=True)
+    _view_name = Unicode("FormView").tag(sync=True)
+    _view_module = Unicode(_module_name).tag(sync=True)
+    _view_module_version = Unicode(__version__).tag(sync=True)
+
+    value = Unicode("Hello World").tag(sync=True)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.on_msg(self._handle_message)
+
+    def _handle_message(self, widget, content, buffers):
+        if "method" in content:
+            method = content["method"]
+            if method == "submit_form":
+                form_data = content["data"]
+                self._process_form_data(form_data)
+
+    def _process_form_data(self, form_data):
+        # Process the form data received from the frontend
+        dropdown_value = form_data.get("dropdown")
+        port_value = form_data.get("port")
```

Comment:
Is this an actual port?  How does the input vary with port? Also, getting some error in the example notebook:
<img width="637" alt="Screenshot 2023-05-29 at 9 41 46 AM" src="https://github.com/ploomber/jupysql-plugin/assets/29096135/9e8de879-9717-40f0-b1ae-4be4d9e61335">

Reply:
It can also show a clearer message